generators/generator-bot-conversational-core/generators/app/templates/language-generation/en-us/botName.en-us.lg

```diff
@@ -10,4 +10,13 @@
 - Sorry, I didn't get that
 - I'm not sure I understand. Can you please try again?
 - Hmm, I don't understand. Can you try to ask me in a different way. 
-- I didn't get that. Would you mind rephrasing and try it again.
\ No newline at end of file
+- I didn't get that. Would you mind rephrasing and try it again.
```

Comment:
nit: new line after

Reply:
This is as generated by composer, I can add it but if we update the template we may end up with the same if we just copy the lg.